dist/docker/redhat/README.md

```diff
@@ -31,9 +31,9 @@ Do the following in the top-level Scylla source directory:
 
 1. Build your own Scylla in whatever build mode you prefer, e.g., dev.
 
-2. Run `./reloc/build_reloc.sh --mode dev`
+2. Run `ninja build/dev/dist/tar/scylla-package.tar.gz`
 
-3. Run `./reloc/build_rpm.sh --reloc-pkg build/dev/scylla-package.tar.gz`
+3. Run `./reloc/build_rpm.sh --reloc-pkg ninja build/dev/dist/tar/scylla-package.tar.gz`
```

Comment:
This seems unlikely to work.

Reply:
btw, better to put the instructions for building rpm with ninja where.

Reply:
Fixed.